pkg/kubelet/container/helpers.go

```diff
@@ -45,7 +45,8 @@ type HandlerRunner interface {
 // RuntimeHelper wraps kubelet to make container runtime
 // able to get necessary informations like the RunContainerOptions, DNS settings, Host IP.
 type RuntimeHelper interface {
-	GenerateRunContainerOptions(pod *v1.Pod, container *v1.Container, podIP string) (contOpts *RunContainerOptions, cleanupAction func(), err error)
+	// TODO (aramase): as podIPs are populated for single stack and dual stack cluster, pass only podIPs.
```

Comment:
I don't think the `Pod.Status.PodIP` will be ever removed, so that TODO will never be done.

@thockin..

Reply:
Removed the TODO.